ui/src/message_popup/popup_question.jsx

```diff
@@ -118,19 +154,29 @@ export default React.createClass({
           student={student}
           question={question}
         />
-        {showResponse && <ResponseRenderer
-          scaffolding={scaffolding}
-          question={question}
-          limitMs={limitMs}
-          onResponseSubmitted={this.onResponseSubmitted}
-          onLogMessage={this.logData}
-        />}
+        {allowResponding && this.renderResponse()}
```

Comment:
For some reason, when the regular text response is rendered, the style is broken. The style gets fixed when the user changes the text in the textfield.

<img width="375" alt="screen shot 2016-08-17 at 3 57 11 pm" src="https://cloud.githubusercontent.com/assets/1490352/17751041/4f27031a-6493-11e6-9801-e076ab37316b.png">
vs.
<img width="373" alt="screen shot 2016-08-17 at 3 57 21 pm" src="https://cloud.githubusercontent.com/assets/1490352/17751053/56f61a2c-6493-11e6-98f9-9e22a9bb2f2b.png">

I ran into a similar issue earlier with Question Authoring but I'm not too sure if it is the same. I was updating the state that belonged in the textfield shortly after it was created and even though the textfield had a paragraph of text, it showed as a single line until edited. Even if it is the same problem, I just changed the code so that I wouldn't have to update the state so I never really found the solution. It's possible that it is also an issue with Material UI but I'm not sure.